scripts/ci/ensure-all-node-requirements-available.sh

```diff
@@ -31,7 +31,7 @@ PACKAGE_JSON=package.json
 # workaround for path-length bug in jq that only affects Windows https://github.com/stedolan/jq/issues/1155
 NODE_MODULES=($(cat "$HERE/builtin-modules.json" | jq -r '.[]'))
 NPM_MODULES=($(jq -r '.dependencies | keys | .[]' "$PACKAGE_JSON"))
-NPM_OPTIONAL_MODULES=($(jq -r '.optionalDependencies | keys | .[]' "$PACKAGE_JSON"))
+NPM_OPTIONAL_MODULES=($(jq -r '.optionalDependencies // {} | keys | .[]' "$PACKAGE_JSON"))
```

Comment:
Probably worth adding this additional syntax to the lines above and below too? (for extra robustness)